pkg/storageops/aws/aws.go

```diff
@@ -216,24 +176,31 @@ func (s *ec2Ops) waitAttachmentStatus(
 	return outVol, nil
 }
 
+func (s *ec2Ops) Name() string { return "aws" }
+
 func (s *ec2Ops) ApplyTags(
-	v *ec2.Volume,
+	v interface{},
 	labels map[string]string,
 ) error {
+
+	vol := v.(*ec2.Volume)
+
 	req := &ec2.CreateTagsInput{
-		Resources: []*string{v.VolumeId},
+		Resources: []*string{vol.VolumeId},
 		Tags:      s.tags(labels),
 	}
 	_, err := s.ec2.CreateTags(req)
 	return err
 }
 
 func (s *ec2Ops) RemoveTags(
-	v *ec2.Volume,
+	v interface{},
 	labels map[string]string,
 ) error {
+	vol := v.(*ec2.Volume)
```

Comment:
vol, ok := v.(*ec2.Volume)
change all the typecasts in this file.

Reply:
Changed API in incremental